translations/strings.yaml

```
@@ -422,6 +424,9 @@ crashreporter:
   dontSendButton:
     value: No thanks
     comment: User does not agree to send a crash report.
+  crashOptInDialogDescription:
+    value:  Don't show this again.  Auto-send crash reports to Mozilla.
```

```suggestion
    value:  Don't show this again. Auto-send crash reports to Mozilla.
```

Drop the double space after punctuation.

Does `Auto-send` come from our Content Team? I've never seen it before, it's normally spelled out.

---

Auto-send was in the figma, but I've changed it.  I think Autosend as one word is the correct usage.